### PR TITLE
docs(faq): play2 instances MTA-2160

### PR DESCRIPTION
**Pull request by bene2k1**
### Description

Add FAQ for PLAY2 Instances

### CODE REVIEW

File: faq/play2-instances.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  description: Discover Scaleway PLAY2 Instances.
+content:
+  h1: PLAY2 Instances
+  paragraph: Discover Scaleway PLAY2 Instances.
+image: play2-instances.webp
+---
+
+## What are PLAY2 Instances?
+Scaleway’s latest generation PLAY2 Instances are designed for testing and development purposes.
+PLAY2 Instances are the second generation of Development Instances after DEV1. PLAY2 Instances are small and agile with adequate performance
```
Comment (RoRoJ) [inline faq/play2-instances.mdx:13]: I don't think we should say "adequate", it doesn't sound right. Maybe "good"? Or else "...are small and agile and meet performance requirements for standard testing and development scenarios". ?

File: faq/play2-instances.mdx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+PLAY2 instances are ideal for bootstrapping your developments and running your tests. Once done, you can rely on the complete Scaleway ecosystem to push your cloud-native applications into production on PRO2 Instances or ENT1
```
Comment (RoRoJ) [inline faq/play2-instances.mdx:14]:
```suggestion
PLAY2 Instances are ideal for bootstrapping your developments and running your tests. Once you're done, you can take advantage of the complete Scaleway ecosystem to push your cloud-native applications into production on PRO2 or ENT1 Instances. 
```

File: faq/play2-instances.mdx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## What are PLAY2 Instances?
+Scaleway’s latest generation PLAY2 Instances are designed for testing and development purposes.
+PLAY2 Instances are the second generation of Development Instances after DEV1. PLAY2 Instances are small and agile with adequate performance
+PLAY2 instances are ideal for bootstrapping your developments and running your tests. Once done, you can rely on the complete Scaleway ecosystem to push your cloud-native applications into production on PRO2 Instances or ENT1 
+Scaleway’s PLAY2 Instances rely on the AMD EPYC™ 7003 Series processor.
+
+## What are the benefits to expect from PLAY2 Instances?
+The available PLAY2 configurations range from 1vCPU to 4vCPU with 2GB RAM per vCPU.
+PLAY2 Instances leverage Block Storage and enhance your data security and resiliency with features such as snapshots and 3x replication of the Instances. With a fast boot time, PLAY2 instances are ideal for continuous integration and deployment workflows.
```
Comment (RoRoJ) [inline faq/play2-instances.mdx:19]:
```suggestion
PLAY2 Instances leverage Block Storage and enhance your data security and resiliency with features such as snapshots and 3x replication of the Instances. With a fast boot time, PLAY2 Instances are ideal for continuous integration and deployment workflows.
```

File: faq/play2-instances.mdx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+PLAY2 Instances are fully compatible with our ecosystem: they work with Object Storage, VPC, Databases, Managed Kubernetes, etc.
+
+## Where are my PLAY2 Instances located?
+PLAY2 Instances are available in all regions: Paris (France), Amsterdam (Netherlands), and Warsaw (Poland).
+
+## Which Linux distributions are available for PLAY2 Instances?
+We provide a wide range of different Linux distributions and InstantApps for Instances. PLAY2 Instances work with a selection of the latest, most-acclaimed images, including Ubuntu, Debian, CentOS and Fedora.
```
Comment (RoRoJ) [inline faq/play2-instances.mdx:26]:
```suggestion
We provide a wide range of different Linux distributions and InstantApps for Instances. PLAY2 Instances work with a selection of the latest, most popular images, including Ubuntu, Debian, CentOS and Fedora.
```

**Review by RoRoJ — APPROVED** [review]

File: faq/play2-instances.mdx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+PLAY2 Instances are fully compatible with our ecosystem: they work with Object Storage, VPC, Databases, Managed Kubernetes, etc.
+
+## Where are my PLAY2 Instances located?
+PLAY2 Instances are available in all regions: Paris (France), Amsterdam (Netherlands), and Warsaw (Poland).
+
+## Which Linux distributions are available for PLAY2 Instances?
+We provide a wide range of different Linux distributions and InstantApps for Instances. PLAY2 Instances work with a selection of the latest, most popular images, including Ubuntu, Debian, CentOS and Fedora.
+
+## Can I migrate from my first generation DEV1 Instance to a PLAY2 Instance?
+Yes. Refer to the [Migrating Gen. 1 to Gen. 2 Instances](/compute/instances/api-cli/migrating-gp-to-pro/ ) page.
```
Comment (ldecarvalho-doc) [inline faq/play2-instances.mdx:29]:
```suggestion
## Can I migrate from my first generation DEV1 Instance to a PLAY2 Instance?
Yes. Refer to the [Migrating Gen. 1 to Gen. 2 Instances](/compute/instances/api-cli/migrating-gp-to-pro/) page.
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**